DistroLauncher-Tests/DistroLauncher-Tests/algorithms_tests.cpp

```diff
@@ -63,4 +96,4 @@ TEST(UpgradePolicyTests, Concat)
     std::filesystem::path example_file{L"/home/fox/documents/example.json"};
     auto with_path = concat(L"diff ", example_file, L" ", example_file.wstring()); // Only first one to be quoted
     ASSERT_EQ(with_path, LR"(diff "/home/fox/documents/example.json" /home/fox/documents/example.json)");
-}
+}
```

Comment:
That remove-newline-at-eof-thingy is really annoying. We'll soon need to implement a way to avoid it.